### PR TITLE
IE font fix 

### PR DESCRIPTION
**Pull request by danji90**
# How to

The SBB fonts loaded into TrafimageMaps used to use the same name as the font file as the font-family, differing from the SBB standard, potetially causing problems when SBB enbeds Trafimage into their websites. The loadFont function was simplified and now takes an additional argument for the font-family so the name is the same as the SBB standard.

Test the font-family in the netlify [review app](https://deploy-preview-709--dev-trafimage-maps.netlify.app), which should now be 'SBBWeb Roman' instead of 'SBBWeb-Roman' 

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] IE11 tested.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

**Comment by oterral** [comment]
@danji90 please takes time to set up properly the PR.

File: src/WebComponent.scss [hunk 1]
```diff
@@ -1,19 +1,19 @@
-@mixin loadFonts($paths...) {
+@mixin loadFonts($fontFamily, $paths...) {
   @each $family in $paths {
```
Comment (oterral) [inline src/WebComponent.scss:2]: this loop seems useless now

Comment (danji90) [reply src/WebComponent.scss]: tbh, I didn't see the point of the loop before this update either, it's probably been useless for a while. But sometimes I don't question stuff enough. Will try to do that more

**Comment by oterral** [comment]
@danji90  don't forget to set the labels next time hotfix or a release

**Pull request merged**